File: native/macos/CustomApp.swift
import Cocoa

class CustomApp {
    init() {
        let app = App.getApp()
        app.client.addCustomFunction("testRequestEventName") {
            (args: [Any?]) in
            let bool = args[0] as! Bool
            let float = args[1] as! CGFloat
            let string = args[2] as! String
            app.client.pushEvent("testResponseEventName", args: [
                !bool, float * 2, string.uppercased(), args[3]
            ])
        }
    }
}

File: native/android/CustomApp.java
package io.neft.customapp;

import io.neft.App;
import io.neft.utils.Consumer;

public class CustomApp {
    public CustomApp() {
        final App app = App.getInstance();
        app.getClient().addCustomFunction("testRequestEventName", new Consumer<Object[]>() {
            @Override
            public void accept(Object[] args) {
                boolean bool = (Boolean) args[0];
                float number = (Float) args[1];
                String string = (String) args[2];
                app.getClient().pushEvent(
                        "testResponseEventName",
                        !bool, number * 2, string.toUpperCase(), args[3]
                );
            }
        });
    }
}
